packages/panna-sdk/src/core/utils/utils.ts

```diff
@@ -207,3 +209,78 @@ export const accountBalanceInFiat = async function (
 
   return result;
 };
+
+/**
+ * Get the total fiat value of multiple tokens and individual token balances
+ * @param params - Parameters for getting multiple account balances
+ * @returns Total value and individual token balances with fiat values
+ * @throws Error if address is invalid or if any token fetch fails
+ * @example
+ * ```ts
+ * // Get portfolio value for multiple tokens
+ * const result = await accountBalancesInFiat({
+ *   client: pannaClient,
+ *   address: '0x...',
+ *   chain: lisk,
+ *   tokens: [
+ *     {}, // Native token
+ *     { address: '0x...' }, // USDC
+ *     { address: '0x...' }  // DAI
+ *   ],
```

Comment:
@matjazv @ishantiw Why the nested object? It can be a simple string array.
Also, maybe, we should return the native token info by default? Or, explicitly export and enforce `0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee` to represent the native token?

```suggestion
 *   tokenAddresses: [
 *     '0x...', // USDC
 *     '0x...'  // DAI
 *   ],
```
or
```suggestion
 *   tokenAddresses: [
 *     '0xeee...', // Native token
 *     '0x...', // USDC
 *     '0x...'  // DAI
 *   ],
```

Reply:
Done.